libs/platform/ebpf_platform.h

```diff
@@ -1326,6 +1326,18 @@ extern "C"
     void
     ebpf_leave_critical_region();
 
+    /**
+     * @brief Convert the provided UTF 8 string into a 16-bit unicode string.
+     *
+     * @param[in] input String to convert.
+     * @param[out] output Converted string.
+     * @retval EBPF_SUCCESS The conversion was successful.
+     * @retval EBPF_NO_MEMORY Unable to allocate resources for the conversion.
+     * @retval EBPF_INVALID_ARGUMENT Unable to convert the string.
+     */
+    ebpf_result_t
+    ebpf_utf8_string_to_unicode(_In_ const ebpf_utf8_string_t* input, _Outptr_ wchar_t** output);
```

Comment:
The name "unicode" is a misnomer, although it is consistent with Windows' misnaming.   All UTF encodings (including UTF-8, UTF-16LE, UTF16-BE, etc.) are Unicode.   Windows often misuses the term "Unicode" when it means UTF-16LE.  We should try to be clearer in my opinion.

Reply:
Because it is consistent with Windows' inaccurate usage, I think it is ok as is now that lines 1332-1335 accurately document the actual meaning.  Hence resolving this conversation.